index.html

```diff
@@ -264,14 +245,19 @@ <h3>
           cost.&rdquo;
         </p>
         <div class="signature">
-          <div class="name">Bassam Tabbara</div>
-          <div class="title">CEO and Founder of Upbound</div>
-          <a href="https://upbound.io"><img src="{{ "/images/upbound.svg" | relative_url }}"/></a>
+          <div class="exec-img">
+            <img srcset="./images/bassam.jpg 1x, ./images/bassam@2x.jpg 2x, ./images/bassam@3x.jpg 3x" alt="Bassam Tabbara" />
```

Comment:
I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper